decidim-initiatives/lib/decidim/api/initiative_api_type.rb

```diff
@@ -17,7 +17,7 @@ class InitiativeApiType < Decidim::Api::Types::BaseObject
       field :minimum_committee_members, GraphQL::Types::Int, "Minimum of committee members", null: true
       field :validate_sms_code_on_votes, GraphQL::Types::Boolean, "Add SMS code validation step to signature process", null: true
       field :undo_online_signatures_enabled, GraphQL::Types::Boolean, "Enable participants to undo their online signatures", null: true
-      field :promoting_comittee_enabled, GraphQL::Types::Boolean, "If promoting committee is enabled", method: :promoting_committee_enabled, null: true
+      field :promoting_committee_enabled, GraphQL::Types::Boolean, "If promoting committee is enabled", method: :promoting_committee_enabled, null: true
```

Comment:
this might require some extra stuff?

Reply:
I have checked, and no further occurrences .